Static/AnalysisStructured.hs

```diff
@@ -596,7 +597,17 @@ anaSpecAux conser addSyms lg libEnv ln dg nsig name opts eo sp rg = case sp of
                    (replaceAnnoted sp2' asp2)
                    pos, nsig3, udg3)
   Combination (Network cItems eItems _) pos -> adjustPos pos $ do
-    let getNodes (cN, cE) cItem = let
+    let (cNodes', cEdges') = networkDiagram dg cItems eItems
+    (ns, dg') <- insertColimitInGraph libEnv dg cNodes' cEdges' name
+    return (sp, ns, dg')
+  _ -> fail $ "AnalysisStructured: " ++ show (prettyLG lg sp)
+
+networkDiagram :: DGraph
+                        -> [LABELED_ONTO_OR_INTPR_REF]
+                        -> [IRI]
+                        -> ([Node], [(Node, Node, DGLinkLab)])
+networkDiagram dg cItems eItems = let 
+        getNodes remove (cN, cE) cItem = let
```

Comment:
...however, even for such local functions, documentation with their purpose would be helpful.